### ISSUE TITLE
Add a vignette on Custom Split Functions

### ISSUE THREAD

**Issue by gmbecker** [Roche/rtables#378]
# Reporting an Issue with rtables

Please briefly describe your problem and, when relevant, the output you expect.
Please also provide the output of `utils::sessionInfo()` or
`devtools::session_info()` at the end of your post.

If at all possible, please include a [minimal, reproducible
example](https://stackoverflow.com/questions/5963269/how-to-make-a-great-r-reproducible-example).
The `rtables` team will be much more likely to resolve your issue if they are
able to reproduce it themselves locally.

Please delete this preamble after you have read it.

---

your brief description of the problem

```r
library(rtables)

# your reproducible example here

```

**Comment by StefanThoma** [comment]
Do you mean a vignette that explains how to use the `split_..._by_cutfun()` functions?

**Comment by gmbecker** [comment]
Yes, this vignette would cover the concept of custom split functions (cases of `split_*_by` where the user specifies `split_fun` directly), what arguments they can take, what types of things can be done with them, etc. See the relevant section of our workshop https://docs.google.com/presentation/d/1t0098eh1b8_FaFfRoD7jhP1nxhLmjrJeD5X5MkmKC48/edit#slide=id.g17317f31d49_0_111 as well in terms of content that would go in this vignette

**Comment by wwojciech** [comment]
Split functions: https://roche.github.io/rtables/main/reference/split_funcs.html

**Comment by Melkiades** [comment]
That is in reference as it is this:
> I think he means something to expand and integrate `?custom_split_funs`

This issue wants it to connect everything in a vignette.

**Comment by wwojciech** [comment]
> That is in reference as it is this:
> 
> > I think he means something to expand and integrate `?custom_split_funs`
> 
> This issue wants it to connect everything in a vignette.

Understood, thanks @Melkiades !

**Comment by wwojciech** [comment]
Blocked until `make_split_fun` function will be available.
